content/src/fieldConfig/profile.json

```diff
@@ -101,13 +102,14 @@
       "taxPin": {
         "default": {
           "header": "Tax PIN",
-          "description": "What is your `Tax PIN|tax-pin`?",
+          "description": "What is your `Tax PIN|tax-pin` ?",
           "errorTextRequired": "Tax PIN must be 4 digits."
         }
       },
       "businessName": {
         "default": {
           "header": "Business Name",
+          "headerContextualInfo": "",
```

Comment:
So is the intent of the story to add the *ability* for any profile label to have a contextual info link? If so, I imagine we want to add this line to pretty much every field, right? Like directly above for example, wouldn't we want to add this same empty-string line (and the corresponding dropdown in config.yml) - otherwise, how will the content team add a contextual info to Tax PIN if they decide that's what they want to do?

Reply:
Yes, we do I thought I got that one but I clearly have missed it